src/components/NavBar/index.tsx

```diff
@@ -35,38 +36,50 @@ const NavBar = () => {
           </h2>
         </div>
       )}
-      <ul className={styles.navList}>
-        {user && (
-          <li>
-            <Link href="/editor" className={styles.navLink}>
-              {capitalizeWord(t('gotomain'))}
-            </Link>
-          </li>
-        )}
+      <div
+        className={classNames(styles.burgerBtn, isOpen ? styles.open : '')}
+        onClick={() => setIsOpen(!isOpen)}
```

Comment:
`const toggleOpen = useCallback(() => { setIsOpen((prev) => !prev }, [])

onClick={toggleOpen }`

Reply:
will be fixed/refactor in following commits